### ISSUE TITLE
Code examples with language set to `typescript jsx` does not render the example

### ISSUE THREAD

**Issue by okonet** [styleguidist/react-styleguidist#1540]
**Current behavior**

The code that uses `typescript jsx` (this is what WebStorm requires to render TypeScript with JSX), 

```
```typescript jsx
<Box p={3} m={2} width={1 / 4} bg="primary">
  Box content
</Box>
```
```

doesn't render the example. It only renders the static code block.

**Expected behavior**

It should render the example.

**Comment by okonet** [comment]
Same with just `typescript`

**Comment by sapegin** [comment]
I think we should report `typescript jsx` to WebStorm — looks like they use some unique names, incompatible with other tools. It doesn't work in Visual Studio Code either.

I think `typescript` and `ts` don't support Markdown, and the correct is `tsx`.

I think we can support WebStorm custom languages too, [look at “Adding aliases”](https://prismjs.com/extending.html#creating-a-new-language-definition):

```js
Prism.languages.foo = Prism.languages['foo-bar'];
```

This should work in [highlightCode.ts](https://github.com/styleguidist/react-styleguidist/blob/master/src/loaders/utils/highlightCode.ts).

**Comment by okonet** [comment]
Done: https://youtrack.jetbrains.com/issue/WEB-44208
